client.go

```diff
@@ -0,0 +1,143 @@
+package plank
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/evergreen-ci/utility"
+	"github.com/pkg/errors"
+)
+
+const defaultBaseURL = "https://logkeeper2.build.10gen.cc"
```

Comment:
Let's make this an env variable so it can be used locally.

Reply:
That's the default variable, in most cases the base URL would get passed in explicitly with the options struct. So for Evergreen this would most likely come from the settings (app config).

Reply:
ok cool I just wanted to confirm that this will work in different environments that makes sense to me.